### PR TITLE
https://github.com/MoveOnOrg/Spoke/issues/24

### PR DESCRIPTION
**Pull request by sandramchung**
https://github.com/MoveOnOrg/Spoke/issues/24 Creates a hidden link on the homepage to take you to organization creation via the creation of an invitation link that redirects you to login. Better than the current method which is to manually create an invitation in rethinkdb and construct a link from the ID. To test, check out this branch, logout of your Spoke instance if you're logged in, go to the homepage (localhost:3000) and mouse over the bottom of the green box to see the hidden link.

### CODE REVIEW

**Comment by lady3bean** [comment]
I'm running into the proxy error again, whether on master or checkout out to this PR. But I think that's a problem on my end. The code looks good to me. Much invite!

File: src/containers/Home.jsx [hunk 1]
```diff
@@ -101,5 +118,26 @@ const mapQueriesToProps = () => ({
   }
 })
 
+const mapMutationsToProps = () => ({
+  // createCampaign: (campaign) => ({
```
Comment (mathemagica) [inline src/containers/Home.jsx]: probably need to add this back in so the rest of the campaign admin page works again

File: src/server/api/schema.js [hunk 2]
```diff
@@ -139,6 +139,11 @@ const rootSchema = `
     userId: String
   }
 
+  input InviteInput {
+    id: String
```
Comment (mathemagica) [inline src/server/api/schema.js:143]: did we need these in the input object? we're not using them, so worth testing whether we can remove them and have the input obj be empty

File: src/containers/Home.jsx [hunk 3]
```diff
@@ -34,6 +34,23 @@ class Home extends React.Component {
     orgLessUser: false
   }
 
+  handleOrgInviteClick = async (e) => {
+    e.preventDefault()
+    const newInvite = await this.props.mutations.createInvite({
+      id: 'cats',
```
Comment (mathemagica) [inline src/containers/Home.jsx]: do we need these inputs? we're not using them, so worth testing whether everything still works when you remove them

Comment (sandramchung) [reply src/containers/Home.jsx]: An empty object definitely doesn't work. {is_valid = true} does work, though - probably because that's the only required attribute in the model definition.

File: src/containers/Home.jsx (hunk 1 above)
Comment (sandramchung) [reply src/containers/Home.jsx]: Campaign admin is reverted to original - just forgot to delete the campaign mutation after copy pasting this part here in the homepage

File: src/server/api/schema.js (hunk 2 above)
Comment (sandramchung) [reply src/server/api/schema.js:143]: Empty input object throws and error. Just {is_valid: true} works, probably bc that's the only required attribute on the invite model https://github.com/MoveOnOrg/Spoke/pull/40/commits/36843534f9600f16d7d1f96d75c7106cc328b7a7

File: src/containers/Home.jsx (hunk 1 above)
Comment (sandramchung) [reply src/containers/Home.jsx]: https://github.com/MoveOnOrg/Spoke/pull/40/commits/40decfb0352d10bd84db89d1f1ceb0fb19745ce1

File: src/server/api/schema.js (hunk 2 above)
Comment (mathemagica) [reply src/server/api/schema.js:143]: 👍

**Comment by mathemagica** [comment]
👍
